src/scripts/main.js

```diff
@@ -1,3 +1,145 @@
 'use strict';
 
-// write code here
+const doc = {
+  orderASC: false,
+  cities: ['Tokyo', 'Singapore', 'London',
+    'New_York', 'Edinburgh', 'San_Francisco'],
```

Comment:
```suggestion
  cities: [
    "Tokyo",
    "Singapore",
    "London",
    "New_York",
    "Edinburgh",
    "San_Francisco",
  ],
```

Reply:
This is a controversial issue...
 Let's say we have an array with 300 single words or more, not seven. 
So, should we really put it on 300 separate lines ??

Maybe this whould do:

  cities: [
     'Tokyo', 'Singapore', 'London', 'New_York', 'Edinburgh', 'San_Francisco'
  ],

Reply:
Yes, we would)
But on real projects you do not enter so many values manually, but get them from the server.

Reply:
So actually you mean that we not going to... any way :))